### ISSUE TITLE
Exporting Theme bug

### ISSUE THREAD

**Issue by aliobjects** [material-foundation/material-theme-builder#325]
**Describe the bug**
Why when I export the theme files it export the M3 default theme not my customized theme

**Desktop (please complete the following information):**
 - OS: [e.g. macebook]
 - App [figma]
 - Version [24.1.16]
![Group 1321317490](https://github.com/material-foundation/material-theme-builder/assets/150593894/b2ef5596-186e-48ae-98af-052d43cba567)

**Comment by rondryrrick** [comment]
Yeah, I have the same question for them, in the previous version of the plugin that was fixed BUT it exported only 20+ colors))) In this update, it contains all the necessary colors but you can't export them 🥲

**Comment by aliobjects** [comment]
I hope they could fix it, it's really annoying to handoff the colors to the dev team manually in unprofessional way and consuming time

**Comment by ummthomas** [comment]
Anyone found a workaround for this? Need to export as JSON to hand off to Eng. Any other plugins being used to do this well?
